portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,54 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/message")
+@WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
+    comments.add(
+        "A ship in port is safe, but that is not what ships are for. "
+            + "Sail out to sea and do new things. - Grace Hopper");
+    comments.add("They told me computers could only do arithmetic. - Grace Hopper");
+    comments.add("A ship in port is safe, but that's not what ships are built for. - Grace Hopper");
+    comments.add("It is much easier to apologise than it is to get permission. - Grace Hopper");
+    comments.add("If you can't give me poetry, can't you give me poetical science? - Ada Lovelace");
+    comments.add("I am in a charming state of confusion. - Ada Lovelace");
+    comments.add(
+        "The Analytical Engine weaves algebraic patterns, "
+            + "just as the Jacquard loom weaves flowers and leaves. - Ada Lovelace");
+    comments.add(
+        "Sometimes it is the people no one can imagine anything of "
+            + "who do the things no one can imagine. - Alan Turing");
+    comments.add("Those who can imagine anything, can create the impossible. - Alan Turing");
```

Comment:
These are beautiful! :)

Reply:
I can't take credit. I took them from the example!

Reply:
They will be changed to real user comments in the next step of the walkthrough

Reply:
From Effective Java:

---

### Favor static member classes over nonstatic
A nested class is a class defined within another class. A nested class should exist
only to serve its enclosing class. If a nested class would be useful in some other
context, then it should be a top-level class. There are four kinds of nested classes:
static member classes, nonstatic member classes, anonymous classes, and local
classes. All but the first kind are known as inner classes. This item tells you when
to use which kind of nested class and why.
A static member class is the simplest kind of nested class. It is best thought of
as an ordinary class that happens to be declared inside another class and has
access to all of the enclosing class’s members, even those declared private. A
static member class is a static member of its enclosing class and obeys the same
accessibility rules as other static members. If it is declared private, it is accessible
only within the enclosing class, and so forth.
One common use of a static member class is as a public helper class, useful
only in conjunction with its outer class. For example, consider an enum describing
the operations supported by a calculator (Item 34). The Operation enum should
be a public static member class of the Calculator class. Clients of Calculator
could then refer to operations using names like Calculator.Operation.PLUS and
Calculator.Operation.MINUS.
Syntactically, the only difference between static and nonstatic member classes
is that static member classes have the modifier static in their declarations.
Despite the syntactic similarity, these two kinds of nested classes are very different. Each instance of a nonstatic member class is implicitly associated with an
enclosing instance of its containing class. Within instance methods of a nonstatic
member class, you can invoke methods on the enclosing instance or obtain a reference to the enclosing instance using the qualified this construct [JLS, 15.8.4]. If an
instance of a nested class can exist in isolation from an instance of its enclosing
class, then the nested class must be a static member class: it is impossible to create
an instance of a nonstatic member class without an enclosing instance.
The association between a nonstatic member class instance and its enclosing
instance is established when the member class instance is created and cannot be
modified thereafter. Normally, the association is established automatically by
invoking a nonstatic member class constructor from within an instance method of
the enclosing class. It is possible, though rare, to establish the association
manually using the expression enclosingInstance.new MemberClass(args).
As you would expect, the association takes up space in the nonstatic member class
instance and adds time to its construction.

Reply:
Thanks for the information! Since in the next step the comments will change I left them as is.